CONTRIBUTING.md

```diff
@@ -247,17 +249,17 @@ You can test Your Installation with the following steps.  These steps assume you
 ## Local testing tools
 FIBO developers are accustomed to using desktop tools like Protégé, TopBraid Composer, VOM, and MagicDraw/CCM.  These tools include a variety of tests that experienced users rely on to determine the correctness of their models. In order to satisfy themselves that the ontologies are correct, they need to be able to test the same configuration of ontologies that will be published (i.e., Production and Development, see above).  
 
-In order to make this possible on their local machine, developers need to be able to load just the Release ontologies (from their current testbed).  This is done by creating a file called **AboutProd.ttl** (also AboutProd.rdf).  Developers working with the pre-release files will want to load all FIBO files; this is done with a file called **AboutDev.ttl** (also AboutDev.rdf).  These "About" files load the same files that are loaded by the publishing process of FIBO (for Prod and Dev, respectively).  
+In order to make this possible on their local machine, developers need to be able to load just the Release ontologies (from their current testbed).  This is done by creating a file called **AboutFIBOProd.rdf**.  Developers working with the pre-release files will want to load all FIBO files; this is done with a file called **AboutFIBODev.rdf**.  These "About" files load the same files that are loaded by the publishing process of FIBO (for Prod and Dev, respectively).  
```

Comment:
Should make clear that this will use the source not the published ontologies. 
Developers should at least locally commit any  changes they made so they will be testing the serialized version.

Reply:
Should tell developers they can also use the MetadataX files to load ontologies for a specific module.

Reply:
Actually not - the MetadataX files talk about the module, but they do not import what you would really want to test various aspects of FIBO, only those things that are in the local dlrectory.  

As a person who constantly loads and tests various aspects of FIBO, I've hand-created a number of 'All" files, which are at the top level in each domain that includes released ontologies.  In FND, there is an AllFND.rdf file, for example, which only loads the released ontologies in FND, and allows one to test them with various reasoners.  Then in BE we have not only AllBE, but AllBE for North America (reference data) and All BE for examples (for the companies whose stocks we are referencing in our SEC use case.  And so forth.  These All files are really very useful, and work as load files for various views of FIBO that are quite helpful during development.  I'd actually like to keep not only these, which cannot be generated / must be hand-crafted, but the relative catalog files that support them in those directories for anyone extending them.

Reply:
The updated instructions should not force users to generate a catalog every time they attempt to use Protege to open FIBO.  This is really horrible from a user-friendly / usability perspective.  

But, there should be a catalog that is current at the fibo top level, a relative catalog in the git working version, that people using Protege to make changes can depend on to be current.

Reply:
And, how would people know when they need to regenerate a catalog?  That seems really unhelpful.  They should be able to expect that the catalog provided for their use in github is dependable and up to date.